### DISCUSSION TITLE
Outline thickness

### DISCUSSION THREAD

**Discussion by astrologiical** [soulhotel/FF-ULTIMA#230] [Q&A] [answered]
Is there any way to change the outline thickness?
![Screenshot 2024-12-18 104924](https://github.com/user-attachments/assets/25b8ff45-efb6-479f-8d13-f24a1497d46a)

**Comment by soulhotel** [comment, marked as answer]
Seems like you already have `ultima.spacing.compact` on `true` right? Are you trying to make the Window outline smaller?

**Comment by astrologiical** [comment]
I did not, thanks for telling me, i must have accidentally clicked that.

**Comment by astrologiical** [comment]
on that note, is it possible to change the color of just the outline?

**Comment by soulhotel** [comment]
> on that note, is it possible to change the color of just the outline?

Do you mean like this?

![image](https://github.com/user-attachments/assets/109774cc-88f5-4bfa-941a-f5f93c037989)

With compact spacing off, yes. As you can see: It wouldn't work too well with the old auto-hiding, because the sidebar is overlayed a bit over the browser content, but with the updated auto-hiding it can be done.

**Reply by astrologiical** [reply]
yes, how would i do that?

**Reply by soulhotel** [reply]
go to your chrome/theme folder. Find nightly-override.css

In chrome/theme/`nightly-override.css` there is a `Browser Content Section` (lines 309-337)

####  (Scroll to line 323 in this code below to see the default setup)

https://github.com/soulhotel/FF-ULTIMA/blob/329cd1908a19cfbc33ff4990e93a9abe3a36ae7c/theme/nightly-override.css#L309-L337

- Lines 323 to 337 handle the border radius and the outline
- for outline, you can change the thickness from `4px` to anything from 1-6px. to keep proper spacing in tact.
- you can change the `outline` color by putting `red` instead of `var(--uc-ultima-window)`
- or youre own color, example: `rgba(150,150,255)`

If you are looking for a specific style, then do that and post what you did, I'll see if I can include it in future updates for ya.

**Comment by astrologiical** [comment]
once i do that do i have to restart firefox?

**Reply by soulhotel** [reply]
Yes. That's the easiest way.

Or you can use the browser toolbox to make changes in live, then save the edits straight to the theme files. Without restarting:

- press: `ctrl`+`shift`+`alt`+`I` to open the browser toolbox
- go to the `Style Editor` Tab
- search for the Nightly file
- find the code, make your changes, click save to file

This way is a lot easier in my opinion, but not everyone knows it exist.

https://github.com/user-attachments/assets/33560f02-a6e8-42fa-ac48-4ef6fa65198a

**Comment by astrologiical** [comment]
![Screenshot 2024-12-18 162429](https://github.com/user-attachments/assets/ef77b55f-82a2-45ce-9a35-9c1feca07409)
what im really asking is if theres an easy way to do this, or if it is a custom style

**Reply by soulhotel** [reply]
That's related to Tiling/Window Managers. The one you see above is specific to Linux.

You can definitely get the outline with [PowerToys](https://learn.microsoft.com/en-us/windows/powertoys/), it's by Microsoft. I used to use this too when I  only used Windows (must have imo).

![image](https://github.com/user-attachments/assets/842075da-bdc6-4be6-b180-2a3730a3b64e)

**Comment by astrologiical** [comment]
ok
